doc_user/UserManual.adoc

```diff
@@ -268,7 +271,7 @@ The OPD Extended module affords the following additional features:
 
 <<<
 
-== 6 Pharmacy ([.underline]##P##harmacy)
+== 4 Pharmacy ([.underline]##P##harmacy)
```

Comment:
There is a typo at line 264:  `possibile` -> `possible`

Reply:
Done, thank you !